utils/docs/template/tmpl/layout.tmpl

```diff
@@ -30,8 +30,8 @@
         <h1><a href="https://threejs.org">three.js</a></h1>
 
         <div id="sections">
-            <span class="selected">docs</span>
-            <a href="/examples/#webgl_animation_keyframes">examples</a>
+            <span class="selected">docs (is this used? it doesn't match the new menu)</span>
+            <a href="/examples/#webgl_animation_keyframes">examples (is this used? it doesn't match the new menu)</a>
         </div>
```

Comment:
> (is this used? it doesn't match the new menu

Yes, this is used in the new documentation based on JSDoc. The file has not been updated with the new navigation (manual/docs), yet. Feel free to do that in your PR. Otherwise, I'll do it in the next days.

FYI: The new docs based on JSDoc will replace the old docs next release. We originally wanted to publish the new docs with `r175`, however, some additional style updates are required, see https://github.com/mrdoob/three.js/pull/30774#issuecomment-2751311481. We head now for `r176`.

Reply:
Ah in that case yeah I need to figure out how to build and preview the JSDoc version of the site to update the PR.

I keep fumbling when navigating to the docs:

Before:

- click on "documentation" on the home page
- start searching to lookup an API doc
- click to view (done)

Currently:

- click on "documentation" on the home page
- start typing, but then realize the API I'm looking for is not found
- Oooh, that's because we're landing on the Manual by default now
- click over to the "docs"
- start searching again to look up an API doc
- click to view (done)

I keep doing this. I'm always wanting to go straight to the search bar to filter for API reference docs. I don't think the Manual will get this same type of usage as often.